src/userobjects/RadialGreensConvolution.C

```diff
@@ -406,3 +360,239 @@ RadialGreensConvolution::threadJoin(const UserObject & y)
   _qp_data.insert(_qp_data.begin(), uo._qp_data.begin(), uo._qp_data.end());
   _convolution.insert(uo._convolution.begin(), uo._convolution.end());
 }
+
+void
+RadialGreensConvolution::insertNotLocalPointNeighbors(dof_id_type node)
+{
+  mooseAssert(!_nodes_to_elem_map[node].empty(), "Node not found in _nodes_to_elem_map");
+
+  for (const auto * elem : _nodes_to_elem_map[node])
+    if (elem->processor_id() != _my_pid)
```

Comment:
NVM - looks like your node_to_elem map is different from the one in `MooseMesh`.